lib/services/internalEngine/index.js

```diff
@@ -624,4 +570,51 @@ class InternalEngine {
   }
 }
 
+function indexName (index, collection, { internal=true } = {}) {
+  const prefix = internal
+    ? '%'
+    : '#';
+
+  return `${prefix}${index}/${collection}`
+}
+
+function extractIndex (esIndex) {
+  return esIndex.slice(1).split('/')[0];
+}
+
+function extractCollection (esIndex) {
+  return esIndex.slice(1).split('/')[1];
+}
+
+function extractIndexes (esIndexes, { internal=true } = {}) {
+  const indexes = new Set()
+
+  for (const esIndex of esIndexes) {
+    if (! internal && esIndex[0] === '%') {
+      continue;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this "continue" statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AContinueStatement)